src/client/voice/networking/VoiceWebSocket.js

```diff
@@ -203,7 +203,7 @@ class VoiceWebSocket extends EventEmitter {
          * @param {Object} data
          * @event VoiceWebSocket#speaking
```

Comment:
Please update the event name too, unless you emit this still somewhere

Reply:
Done (I think?), and committed. Cheers.